src/core/components/array-model.jsx

```diff
@@ -31,15 +33,17 @@ export default class ArrayModel extends Component {
     return <span className="model">
       <ModelCollapse title={titleEl} collapsed={ depth > expandDepth } collapsedContent="[...]">
         [
+          {
+            properties.size ? properties.entrySeq().map( ( [ key, v ] ) => <span key={`${key}-${v}`} style={ propStyle }>
+              <br />{ key }: { String(v) }</span>)
+              : null
+          }
```

Comment:
Note that this removes a wrapping span from the previously generated markup. It makes it consistent with [primitive-model.jsx](https://github.com/swagger-api/swagger-ui/blob/master/src/core/components/primitive-model.jsx#L37) though, and appears to have no visible side effects.